src/Spout/Reader/XLSX.php

```diff
@@ -239,6 +224,58 @@ protected function read()
     }
 
     /**
+     * Returns the (unescaped) cell value associated to the given XML node.
+     *
+     * @param \DOMNode $node
+     * @param \Box\Spout\Common\Escaper\XLSX $escaper
+     * @return string|int|float|bool|null The value associated with the cell (null when the cell has an error)
+     */
+    protected function getCellValue($node, $escaper)
+    {
+        $cellValue = '';
+
+        // Default cell type is "n"
+        $cellType = $node->getAttribute('t') ?: 'n';
+
+        if ($cellType === 'inlineStr') {
```

Comment:
Any way this could be further separated into calling two functions, one for inlineStr, and one for other cell types?